plugins/org.wso2.developerstudio.eclipse.gmf.esb.diagram/src/org/wso2/developerstudio/eclipse/gmf/esb/diagram/custom/deserializer/LogMediatorDeserializer.java

```diff
@@ -109,10 +109,10 @@ public LogMediator createNode(IGraphicalEditPart part, AbstractMediator mediator
 		}
 		
 		
-		if (!StringUtils.isBlank(logMediator.getSeparator())) {
+		if (!logMediator.getSeparator().isEmpty()) {
 			
 			//visualLog.setLogSeparator(logMediator.getSeparator());
-			executeSetValueCommand(LOG_MEDIATOR__LOG_SEPARATOR, logMediator.getSeparator());
+			executeSetValueCommand(LOG_MEDIATOR__LOG_SEPARATOR, logMediator.getSeparator().replace("\n", "\\n").replace("\t", "\\t"));
```

Comment:
exceeds 120 limit?

Reply:
Fixed in : https://github.com/wso2/devstudio-tooling-esb/pull/272